### DISCUSSION TITLE
Achievement Ladder in Website

### DISCUSSION THREAD

**Discussion by armchairrockstar1** [chromiecraft/chromiecraft#7311] [Suggestions]
It would be great to see the players with most achievement points on the website, just like we see top PvP'ers. A top 100 ladder would be great. Thank you.

**Comment by Helias** [comment]
I personally worked on this in past ( https://github.com/Helias/TC-PvEstats/ ) and I did also some special ranking list for the guild.
I would like to restore [this very old tool](https://github.com/Helias/TC-PvEstats/) and use it for chromiecraft, I will consider your request, thanks!

**UPDATE 2026/04/07:** PlayerStats is live now! https://www.chromiecraft.com/en/players-stats/

**Reply by FFabz** [reply]
Hello, any update on this ? the picture in live demo of your very old tool is exactly what we need. I tried to create a python script with the help of chat gpt that would parse the total-points value have this in a shared Gsheet but was unsuccesful (idk how to deal with this i-frame).

**Reply by FFabz** [reply]
Ok I actually was able to get into the I-frame, now just gotta automate is for char name from a csv
![image](https://github.com/user-attachments/assets/fbde50c2-c90e-4d9a-a689-810e2ea26cd0)
e

**Reply by Helias** [reply]
Hi, unfortunately I have no updates about this, but I am happy to see that more users are interested on this ladder

**Reply by FFabz** [reply]
I guess for now we'll use a Gsheet and my script haha

**Reply by Helias** [reply]
> I guess for now we'll use a Gsheet and my script haha

Are you listing the players based on the total sum of Achievement points?

**Reply by FFabz** [reply]
yes I made a sheet here : https://docs.google.com/spreadsheets/d/1q50sfgW-_QindLm5crgegTJHy-JyQcEWiUZdgVLDJ0U/edit?gid=1530138848#gid=1530138848 

List of player is coming from pvpstats, I removed non 70 players

**Reply by Borrek98** [reply]
@FFabz Hey! could you update the list once in a while? would like to see my progress! :D also, do we have some news on this, if its planned on being implemented to the website?

**Reply by FFabz** [reply]
Hi @Borrek98 I'm glad you enjoy my ranking. For now i'll update it once per month since there is little new content, prolly by the end of the week for this month. When Wotlk releases i'll do weekly ones as it will move a lot with all the new content

**Reply by Opalmaster** [reply]
It would be cool to include a column with the number of Feats of Strength (FoS) the player has.

**Comment by Borrek98** [comment]
To have that ladder on the website would be awesome! There are a lot of achievers on Chromie, and it would be another great motivator for them. Also would add another "competitive" feature besides PvP, an alternative for PvE players as well (even more with WoTlk, where majority of the achievements are PvE based)

**Comment by Borrek98** [comment]
Hello everyone! Are there any news on the achievement ladder being introduced to the website in a similar way to PvP stats? I think many people would really appreciate it and could enhance some healthy competition :-) @Helias

**Reply by Helias** [reply]
Yes, there is a news ;-)

Here is the PlayerStats https://www.chromiecraft.com/en/players-stats/ , available through Tools -> PlayersStats
I have resurrected the old pve stats application that I linked; now it's not needed anymore to update the CSV file.

Note: the PlayersStats update itself every hour.

**Comment by BeefSteaker** [comment]
Now that its live, itd be cool to introduce filters. 
Filter by guild so you can compete with guildies, or by class so you can see where you rank in that

**Reply by Helias** [reply]
Thanks!

In the old player stats there were multiple top lists, per guild, per playing time and more
I will collect the ideas here and slowly add more lists in the web app
